### DISCUSSION TITLE
Snap app not shown in launcher

### DISCUSSION THREAD

**Discussion by dev-keshav-raj** [Schneegans/Burn-My-Windows#255] [Q&A] [answered]
after installing any application from snap it not shown in application area why ?
i have to run app by command 

snap run blender 

it shows your path is not added ,but i added path at last of  ./bashrc file 
 export PATH="$PATH:/snap/bin"
export PATH="$PATH:/home/usr-name/snap"

after doing this i done echo $PATH but my added path is not present in the path area .

How i can fix this issue ?
And how i add path correctly

![selection](https://user-images.githubusercontent.com/109474474/208220907-d3acf805-2e71-41fe-829a-cc8efb434d64.png)
![path](https://user-images.githubusercontent.com/109474474/208220963-ef04503b-4058-4753-8417-9b043eae23d2.png)

**Comment by Schneegans** [comment, marked as answer]
I am pretty sure that this is in no way related to Burn-My-Windows...

**Reply by dev-keshav-raj** [reply]
NO it's not
